lib/tasks/evm.rake

```diff
@@ -29,7 +29,10 @@ namespace :evm do
   end
 
   task :update_start do
-    EvmApplication.update_start
+    EvmRakeHelper.with_dummy_database_url_configuration do
```

Comment:
I think this deserves a comment.  Why are all of the other rake tasks loading environment normally while we jump through hoops to ensure environment is loaded with a dummy database url?  I think I understand now, but I won't know when I see this in 2 months.